freqtrade/arguments.py

```diff
@@ -237,13 +240,17 @@ def parse_timerange(text: str) -> Optional[Tuple[List, int, int]]:
                     start = rvals[index]
                     if stype[0] == 'date':
                         start = arrow.get(start, 'YYYYMMDD').timestamp
+                    elif stype[0] == 'timestamp':
```

Comment:
I'm not sure if adding `"timestamp"` as type makes sense as the start/stop values are in epoch anyway (unix timestamp) - so sticking to `"date"` should work the same.

The way you have it currently, it will not work as expected at least in the following code (but there are other segments where `"date"` and `"line"` are used in if/elif checks.

https://github.com/gcarq/freqtrade/blob/81bb128cf70655af7ba579c6d2bda88ae4f74d66/freqtrade/optimize/__init__.py#L154-L158

Reply:
awesome feedback - ill do some more tests.
I think i need it to be something not-date initially to handle the timestamp differently yymmdd or itll be corrupted, but can be given the date stype after.